composer.json

```diff
@@ -11,13 +11,13 @@
         }
     ],
     "require": {
-        "php": ">=5.6.0",
-        "illuminate/support": "5.*",
-        "illuminate/validation": "5.*"
+        "php": "^7.0",
```

Comment:
We test on PHP 7.1+, so should we also require PHP 7.1+?

Reply:
Good catch; Yes, we should!